src/esploader.ts

```diff
@@ -155,8 +155,7 @@ async function magic2Chip(magic: number): Promise<ROM | null> {
       const { ESP8266ROM } = await import("./targets/esp8266.js");
       return new ESP8266ROM();
     }
-    case 0x0addbad0:
-    case 0x0: {
```

Comment:
The previous change seemed right - Magic value for p4 chip seems to be - 0x0addbad0

Reply:
Adding them now

Reply:
@adwait-esp which chip revision do you have? (You can check using esptool.py)

Reply:
Those values are listed in current esptool master. I've tested with Function EV Board but magic value was `0x7039ad9`